index-handlers/src/test/java/no/sikt/nva/nvi/index/SearchNviCandidatesHandlerTest.java

```diff
@@ -150,7 +150,7 @@ void shouldReturnPaginatedSearchResultWithoutQueryParamFilterInIdIfNotGiven() th
     }
 
     @Test
-    void shouldReturnPaginatedSearchResultWithCorrectQueryParamsFilterAndQueryInIdIfGiven() throws IOException {
+    void shouldReturnPaginatedSearchResultWithCorrectQueryParamsInIdIfGiven() throws IOException {
```

Comment:
I would also assert that id contains expected title, like this:
```
@Test
    void shouldReturnPaginatedSearchResultWithCorrectQueryParamsInIdIfGiven() throws IOException {
        mockOpenSearchClient();
        var randomFilter = randomString();
        var randomInstitutions = List.of(randomSiktSubUnit(), randomSiktSubUnit());
        var randomTitle = randomString();
        handler.handleRequest(requestWithInstitutionsAndFilter(randomInstitutions, randomFilter, randomTitle),
                              output, context);
        var response = GatewayResponse.fromOutputStream(output, PaginatedSearchResult.class);
        var paginatedSearchResult = response.getBodyObject(PaginatedSearchResult.class);

        var actualId = paginatedSearchResult.getId().toString();
        var expectedInstitutionQuery = QUERY_PARAM_AFFILIATIONS
                                       + "=" + randomInstitutions.get(0)
                                       + "," + randomInstitutions.get(1);
        var expectedExcludeQuery = QUERY_PARAM_EXCLUDE_SUB_UNITS + "=true";
        var expectedTitleQuery = QUERY_PARAM_TITLE + "=" + randomTitle;

        assertThat(actualId, containsString(QUERY_PARAM_FILTER + "=" + randomFilter));
        assertThat(actualId, containsString(QUERY_PARAM_TITLE + "=" + randomTitle));
        assertThat(actualId, containsString(expectedInstitutionQuery));
        assertThat(actualId, containsString(expectedExcludeQuery));
        assertThat(actualId, containsString(expectedTitleQuery));
    }
```

Reply:
Is already done. See the code above it.